tests/Unit/Plugin/Task/Checkfiles/Extension/CheckfilesPluginTest.php

```diff
@@ -0,0 +1,221 @@
+<?php
+/**
+ * @package     Joomla.UnitTest
+ * @subpackage  Extension
+ *
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Tests\Unit\Plugin\Task\Requests\Extension;
+
+use Joomla\CMS\Application\CMSApplicationInterface;
+use Joomla\CMS\Language\Language;
+use Joomla\Component\Scheduler\Administrator\Event\ExecuteTaskEvent;
+use Joomla\Component\Scheduler\Administrator\Task\Status;
+use Joomla\Component\Scheduler\Administrator\Task\Task;
+use Joomla\Event\Dispatcher;
+use Joomla\Filesystem\Folder;
+use Joomla\Plugin\Task\Checkfiles\Extension\Checkfiles;
+use Joomla\Tests\Unit\UnitTestCase;
+
+/**
+ * Test class for Requests plugin
+ *
+ * @package     Joomla.UnitTest
+ * @subpackage  Requests
+ *
+ * @testdox     The Checkfiles plugin
+ *
+ * @since       __DEPLOY_VERSION__
+ */
+class RequestsPluginTest extends UnitTestCase
+{
+	/**
+	 * Setup
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function setUp(): void
+    {
+		if (!is_dir(__DIR__ . '/tmp'))
+		{
+			mkdir(__DIR__ . '/tmp');
+		}
+
+		$image = imagecreate(200, 200);
+        imagecolorallocate($image, 255, 255, 0);
+        imagepng($image, __DIR__ . '/tmp/test.png');
+        imagedestroy($image);
```

Comment:
```suggestion
		$image = imagecreate(200, 200);
        imagecolorallocate($image, 255, 255, 0);
        imagepng($image, __DIR__ . '/tmp/test.png');
        imagedestroy($image);
```
```suggestion
		$image = imagecreate(200, 200);
		imagecolorallocate($image, 255, 255, 0);
		imagepng($image, __DIR__ . '/tmp/test.png');
		imagedestroy($image);
```
@laoneo You really should adjust your editor so you can see what is a tab and what are spaces, and switch off any auto indentation features. I found this place here now where a mix was used. I haven't checked at other places of this PR. Please do so.